sdk/healthinsights/azure-health-insights-cancerprofiling/src/test/java/com/azure/health/insights/cancerprofiling/CancerProfilingClientTest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.health.insights.cancerprofiling;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypePatientResult;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypeResult;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypeInference;
+
+import com.azure.core.util.serializer.TypeReference;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.util.BinaryData;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Unit tests for {@link CancerProfilingClient}.
+ */
+public class CancerProfilingClientTest extends CancerProfilingClientTestBase {
+
+    private CancerProfilingClient getClient() {
+        return getClientBuilder().buildClient();
+    }
+
+    @Test
+    public void test() {
+        try {
+            testCancerProfilingWithResponse(request -> {
+                BinaryData responseValue = getClient().beginInferCancerProfile(request, new RequestOptions()).waitForCompletion().getValue();
```

Comment:
https://github.com/Azure/azure-sdk-for-java/wiki/Cadl-Java-Quickstart#recordplayback-tests

⚠️ Use TestBase.setPlaybackSyncPollerPollInterval on the SyncPoller instance, if your test involves LRO API.